runtime/commonMain/src/kotlinx/benchmark/SuiteDescriptor.kt

```diff
@@ -1,9 +1,9 @@
 package kotlinx.benchmark
 
 object DefaultDescriptorParameters {
```

Comment:
Could you please move `SuiteDescriptor.outputTimeUnit` and `SuiteDescriptor.mode` default values to this object as well?
Also please make sure the specified defaults are the same in JVM.

Reply:
Absolutely, pushing an update here shortly addressing your concerns.

We might want to consider decoupling the JVM default values from JMH as well, since the JMH defaults are subject to change (though unlikely). https://github.com/Kotlin/kotlinx-benchmark/blob/master/runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt#L17

Reply:
For the time being, I have aligned them with the JMH defaults https://github.com/openjdk/jmh/blob/master/jmh-core/src/main/java/org/openjdk/jmh/runner/Defaults.java.